scripts/mkcloud

```diff
@@ -1326,6 +1398,9 @@ function expand_steps()
                     plain_with_upgrade)
                         runcmds="$runcmds `expand_steps plain` addupdaterepo runupdate `expand_steps _upgrade`"
                     ;;
+                    setup_crowbar_development)
+                        runcmds="$runcmds cleanup prepare setupadmin prepareinstcrowbar instcrowbar crowbar_dev_init crowbar_dev_update crowbar_dev_guard crowbar_dev_install"
```

Comment:
rather call these consecutive steps from a function, than hiding the required chain here

My recommendation would be something like this:

```
function setup_crowbar_development()
{
   source ${mkcloud_lib_dir}/crowbar-dev-lib.sh  # <--- do the fancy magic here
   # ^^^ this sources the following 4 functions:
   safely crowbar_dev_init
   safely crowbar_dev_update
   safely crowbar_dev_guard
   safely crowbar_dev_install
}

@@ function expand_steps()
...
install_crowbar_development)
    runcmds="$runcmds cleanup prepare setupadmin prepareinstcrowbar instcrowbar setup_crowbar_development"
;;
...
```

With this we can install the full stack from 0 to crowbar-dev via 
`mkcloud install_crowbar_development`
And we can run it as separate step via
`mkcloud something ; do $otherthing ; mkcloud setup_crowbar_development`.
And we do not mess with the input fds of all mkcloud runs (which could create issues).